styles/sass/partials/_popup.scss

```diff
@@ -48,3 +48,40 @@
     }
   }
 }
+
+.popup__alert {
+  background-color: rgba($color: $offWhite, $alpha: 1);
+  width: 50%;
```

Comment:
At 50% width, does this popup appear too small in mobile view? If so, I might suggest using clamp here.